README.md

```diff
@@ -251,9 +251,98 @@ Congratulations! You can now send remote push notifications to your device from
+	// This will most likely be the global AppDelegate class.
+	- (enum InAppShowResponse)onNewMessage:(IterableInAppMessage * _Nonnull)message {
+   		// perform custom processing
+   		
+   		// ...
+   		
+    	return InAppShowResponseShow; // or InAppShowResponseSkip
+	}
+
+	// ...
+	
+	// Now set this custom class in IterableConfig
+	IterableConfig *config = [[IterableConfig alloc] init];
+	config.inAppDelegate = self; // or other class implementing the protocol
+	[IterableAPI initializeWithApiKey:@"YOUR API KEY" launchOptions:launchOptions config:config];
+	```
+
+	###### Getting the Local Queue of InApp Messages
+	All inApp messages that arrive from the server are stored in a local queue until they are consumed. IterableSDK exposes `InAppManager` protocol via the `IterableAPI.inAppManager` read-only property to get to locally stored inApp messages. Please be aware that all inApp messages that are shown will be consumed and removed from this queue by default. So you will have to override the default behavior as mentioned above to keep inApp messages around even after they are shown.
+	
+	Swift:
+	
+	```
+	// Get the inApp messages list
+	let messages = IterableAPI.inAppManager.getMessages()
+	
+	// Show an inApp message 
+	IterableAPI.inAppManager.show(message: message)
+	
+	// Show an inApp message without consuming, i.e., not removing it from the queue
+	IterableAPI.inAppManager.show(message: message, consume: false)
+	
+	```	
+	
+	Objective-C:
+	
+	```
+	// Get the inApp messages list
+    NSArray *messages = [IterableAPI.inAppManager getMessages];
+	
+	// Show an inApp message 
+	[IterableAPI.inAppManager showMessage:message];
+	
+	// Show an inApp message without consuming, i.e., not removing it from the queue
+	[IterableAPI.inAppManager showMessage:message consume:NO callbackBlock:nil];
+	
+	```	
+
+	###### When User Clicks a Button in the InApp Message
+	If the clicked `href` in the inApp message is a url (which is the case most of the time), `IterableURLDelegate` is called. If you don't set a custom class for `IterableConfig.urlDelegate` then mobile Safari will be opened with the clicked href url.
+	
+	If the clicked `href` is not a url but a custom action name,  `IterableCustomActionDelegate` is called. Please see example in sample code [here](https://github.com/Iterable/swift-sdk/blob/master/sample-apps/swift-sample-app/swift-sample-app/AppDelegate.swift) to see how to implement `IterableURLDelegate` and `IterableCustomActionDelegate` and handle clicked urls.
+	
+	```
+	let config = IterableConfig()
+	config.urlDelegate = YourCustomUrlDelegate()
+	config.customActionDelegate = YourCustomActionDelegate()
+	```
+	
+	###### Changing the Display Interval Between InApp Messages
+	If you want to change the display interval, i.e., the time delay to show two successive inApp messages, to some value other than 30 seconds, change `IterableConfig.inAppDisplayInterval` to the appropriate value.
```

Comment:
- No indent

Suggestion:

> To customize the time delay between successive in-app messages (default value of 30 seconds), set `IterableConfig.inAppDisplayInterval` to an appropriate value.

Probably good to specify units here—seconds? milliseconds?